src/components/editgrid/EditGrid.js

```diff
@@ -64,7 +64,7 @@ export default class EditGridComponent extends NestedArrayComponent {
       {% util.eachComponent(components, function(component) { %}
         {% if (!component.hasOwnProperty('tableView') || component.tableView) { %}
           <div class="col-sm-2">
-            {{ getView(component, row[component.key]) }}
+            {{ ctx.t(getView(component, row[component.key])) }}
```

Comment:
This is strange that we are calling ```ctx.t``` on the getView method. If I understand correctly, this performs an interpolation on a string that they configure in the row template. So, why can they not just provide the ```ctx.t()``` calls within those templates that they define?  This concerns me regarding performance and bad side effects.

Reply:
hm, don't actually remember for what reason I've done that. You're right, it can be done through templates configuration. Will get rid of this.